Fix universalis query not working in non-inject mode and misc fix

sandtechnology opened the pull request.

sandtechnology committed:
fix: fix querying market items in non-inject mode
Change the time when we query for universalis and delay the dto fire time to fix item id not handle issue

sandtechnology committed:
chore: check the item id when uploading data to universalis
now it mainly blocking market history uploading (since we still does not fix the item id issue when uploading history)

sandtechnology committed:
fix: fix wording and format for MarketBoardRequestItemListingInfo comment

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/thewakingsands/matcha?pullRequest=24) <br/>All committers have signed the CLA.

sandtechnology committed:
feat: refactor MarketBoardItemListingCountDTO event firing
Remove the MarketBoardRequestItemListingInfo and MarketBoardItemListingCount handling, seems MarketBoardItemListingCount not longer work for just sending all zero values

sandtechnology commented:
Still having some problems:
1. Market history uploading not fixed (we don't know where to take its item id).
2. When not seller in sell, matcha will not detected item query in market.

zhyupe commented:
Merged into a separate branch for further modification
